assets/src/scripts/components/hydrograph/index.js

```diff
@@ -162,7 +163,8 @@ const plotSvgDefs = function(elem) {
 
 const timeSeriesLegend = function(elem) {
     elem.append('div')
-        .attr('class', 'hydrograph-container')
+        .classed('timeseries-legend-container', true)
```

Comment:
I don't this class is used anywhere

Reply:
Yeah, I was using it in the css but I am not anymore.